internal/policy/engine.go

```diff
@@ -75,6 +84,27 @@ func (e *Engine) Modules() map[string]*ast.Module {
 	return e.modules
 }
 
+func (e *Engine) LoadPolicy(filename, policy string) error {
```

Comment:
can this parameter be another source than a filename?

Reply:
Maybe just `name`? The thing is that whenever we support tracing this property will be referred to as `filename`, since that's what OPA uses. It's more like a "virtual" filename.

Reply:
OK, lets keep it as filename.